test/Plutip/Contract.purs

```diff
@@ -123,6 +127,11 @@ import Ctl.Internal.Plutus.Types.Value (lovelaceValueOf)
 import Ctl.Internal.Scripts (nativeScriptHashEnterpriseAddress)
 import Ctl.Internal.Test.TestPlanM (TestPlanM)
 import Ctl.Internal.Types.Interval (getSlotLength)
+import Ctl.Internal.Types.TransactionMetadata
```

Comment:
I know we have loads of internal imports already, but lets try and import from the public interface where possible, as it adds another assurance to our API